schema/aws/defaults/infrastructure/virtual-machine.yml

```diff
@@ -86,6 +86,13 @@ specification:
        destination_port_range: "25672"
        source_address_prefix: "10.1.8.0/24"
        destination_address_prefix: "0.0.0.0/0"
+     - name: rabbitmq-exporter
+       description: Allow Prometheus for RabbitMQ's data scrap
```

Comment:
Description causes issues on AWS:
`Error: "ingress.3.description" doesn't comply with restrictions ("^[A-Za-z0-9 \\.\\_\\-\\:\\/\\(\\)\\#\\,\\@\\[\\]\\+\\=\\;\\{\\}\\!\\$\\*]*$"): "7 - Allow Prometheus for RabbitMQ's data scrap"`